apps/web/src/app/dashboard/[name]/overview/page.tsx

```diff
@@ -40,14 +41,18 @@ export default function Dashboard({ params }: DashboardOverviewProps) {
 
   useEffect(() => {
     console.log(organizations);
-    if (!isLoading && (!organizations || !organizations.map((org) => org.name).includes(params.name))) {
+    if (
+      !isLoading &&
+      !organizations?.map((org) => org.name).includes(params.name)
+    ) {
       router.push("/dashboard");
     }
   }, [organizations, router]);
 
   return (
-    (!isLoading && organizations!.map((org) => org.name).includes(params.name) &&
-      params.name) && (
+    !isLoading &&
+    organizations!.map((org) => org.name).includes(params.name) &&
```

Comment:
This is not correct since `useQuery` can return `null` if the query errored out. Add error handling for this.